pytorch_lightning/trainer/connectors/data_connector.py

```diff
@@ -288,7 +288,7 @@ def dataloader(self) -> Union[TRAIN_DATALOADERS, EVAL_DATALOADERS]:
             return self.instance
 
         if isinstance(self.instance, LightningModule):
-            return self.instance.trainer.call_hook(self.name, pl_module=self.instance)
```

Comment:
why did this remove?

`pl_module=self.instance`

Reply:
not sure, I put it back. I wish we didn't need this tho :/